WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -225,8 +225,9 @@ def post_results
         body += "#{record_strs.join(", ")}.  \n" # Trailing spaces for markdown give us a <br>
       end
     end
-    post = Post.new(title: title, body: body, author: current_user, world_readable: true)
-    post_post(post)
+    comp.update(results_posted_at: Time.now)
+    comp.save!
```

Comment:
If there are many places to fix, just fixing the biggest problem(s) would be useful, even if unpublished data leaks out elsewhere.

Asking the results team sounds very sensible, though I don't know how to do that, and wasn't even aware such a team exists.

_(This may be the wrong place to have this discussion)_